.github/stale.yml

```diff
@@ -0,0 +1,49 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - WIP
+  - work in progress
+  - important
+  - feature
+  - bug
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  Thanks for contributing to this issue. As it has been 90 days since the last activity,
+  we are automatically closing the issue in 7 days. This is often because the request was
```

Comment:
Need to adjust this comment as well

Reply:
I've already done this here - acd460f5f463bae4a154b0e9ace6b3f0a1f8cc6e (looks like you were looking at the out of date diff).